src/containers/RegisterContainer.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import SignupComponent from "../components/Auth/Signup";
```

Comment:
@magicmarie I think there is no need for adding component or container to your component names

Reply:
noted